tests/network/checkup_framework/utils.py

```diff
@@ -149,6 +150,7 @@ def compose_configmap_data(
     dpdk_test_target_node=None,
     dpdk_vmgen_container_diskimage=None,
     dpdk_vmtest_container_diskimage=None,
+    verbose=None,
```

Comment:
Well, I do not get it.

Please convince us that we need an option here.
It seems this change was triggered by the need to always have it enabled, so I do not understand why you have this as an option.

Reply:
I want to easily have the option to add verbose. This is useful in case of a failure in the checkup. Why not have it?

Reply:
I am just asking you to reason why this is an option for the caller to decide on.
If you want it enabled, it is not a reason to expose it as an argument.

Reply:
I'll change it to be a enabled and not as an argument.